lib/v2/coindesk/index.js

```diff
@@ -7,7 +7,6 @@ module.exports = async (ctx) => {
 
     const response = await got.get(`${rootUrl}/${channel}`);
```

Comment:
```suggestion
    const response = await got.get(`${rootUrl}/${channel}/`);
```
https://github.com/DIYgod/RSSHub/actions/runs/4395357485/jobs/7697089862#step:11:16